pkg/controller/logcollector/logcollector_controller.go

```diff
@@ -551,20 +563,21 @@ func (r *ReconcileLogCollector) Reconcile(ctx context.Context, request reconcile
 
 	if hasWindowsNodes {
 		fluentdCfg = &render.FluentdConfiguration{
-			LogCollector:    instance,
-			ESSecrets:       esSecrets,
-			ESClusterConfig: esClusterConfig,
-			S3Credential:    s3Credential,
-			SplkCredential:  splunkCredential,
-			Filters:         filters,
-			EKSConfig:       eksConfig,
-			PullSecrets:     pullSecrets,
-			Installation:    installation,
-			ClusterDomain:   r.clusterDomain,
-			OSType:          rmeta.OSTypeWindows,
-			TrustedBundle:   trustedBundle,
-			ManagedCluster:  managedCluster,
-			UsePSP:          r.usePSP,
+			LogCollector:     instance,
+			ESSecrets:        esSecrets,
+			ESClusterConfig:  esClusterConfig,
+			S3Credential:     s3Credential,
+			SplkCredential:   splunkCredential,
+			SysLogCredential: syslogCredential,
```

Comment:
I think we don't need this Syslogcredential at all:
- If the user brings a certificate, we can add the PEM to the TrustedBundle. (TrustedBundle.AddCertificates(...) and certificatemanagement.NewCertificate(...) )
- Then the renderer can simply mount the TrustedBundle in the location that syslog reads from.

This way you can remove a lot of code and continue to follow consistent code practices.

Reply:
Done